returnn/compile.py

```diff
@@ -43,7 +43,7 @@ def __init__(
         :param int eval:
         :param int search:
         :param int log_verbosity: RETURNN log verbosity from 1 (least verbose) to 5 (most verbose)
-        :param str device: optimize graph for cpu or gpu
+        :param str|None device: optimize graph for cpu or gpu
```

Comment:
```suggestion
        :param str|None device: optimize graph for cpu or gpu. If `None`, defaults to cpu for current RETURNN. 
            For any RETURNN version before `cd4bc382`, the behavior will depend on the `device` entry in the
            `returnn_conig`, which will depend on the availability of the GPU if not defined at all. 
```

Reply:
maybe like this?

Reply:
> the `device` entry in the
            `returnn_conig`, which will depend on the availability of the GPU if not defined at all.

Well not the device entry depends on the availability. Therefore

```
            For any RETURNN version before `cd4bc382`, the behavior will depend on the `device` entry in the
            `returnn_conig`, or on the availability of a GPU on the execution host if not defined at all.
```